lib/iris/tests/integration/test_netcdf.py

```diff
@@ -23,13 +23,15 @@
 # importing anything else.
 import iris.tests as tests
 
+from itertools import repeat
```

Comment:
@jswanljung Sorry, but within each import block, the imports should be in alphabetical order, so `itertools` comes after `contextlib` and before `os.path` :smile: